packages/nc-gui/components/smartsheet/header/Cell.vue

```diff
@@ -90,18 +90,20 @@ const onClick = (e: Event) => {
         'self-start': isForm || isSurveyForm,
       }"
     />
-    <div
-      v-if="column"
-      class="name pl-1"
-      :class="{
-        'cursor-pointer pt-0.25': !isForm && isUIAllowed('fieldEdit') && !hideMenu && !isExpandedForm,
-        'cursor-default': isForm || !isUIAllowed('fieldEdit') || hideMenu,
-        '!truncate': !isForm,
-      }"
-      :data-test-id="column.title"
-    >
-      {{ column.title }}
-    </div>
+    <a-tooltip placement="bottom" :title="column.title">
```

Comment:
Use NcTooltip component

Reply:
**Fixed in below commit**

[fix : Used NcTooltip](https://github.com/nocodb/nocodb/pull/7114/commits/9207eda4c98368d9b747fb3ccff4a37b70b60fc7)
[9207eda](https://github.com/nocodb/nocodb/pull/7114/commits/9207eda4c98368d9b747fb3ccff4a37b70b60fc7)